Flaky test: DistSyncOnePhaseWriteSkewTxStateTransferTest fails with bridge view timeout

### Description

`DistSyncOnePhaseWriteSkewTxStateTransferTest` fails intermittently during setup (`createBeforeMethod`) with:

```
java.lang.AssertionError: Timed out waiting for bridge view [NYC-2, LON-1] on DistSyncOnePhaseWriteSkewTxStateTransferTest-NodeAY after 10 SECONDS. Current bridge view is [NYC-2]
```

### Root Cause

In `AbstractXSiteTest.waitForSites()`, the deadline is computed once before iterating over all site masters. If the first site master (LON) consumes most of the 10-second timeout to see the bridge view converge, subsequent site masters (NYC) get little or no remaining time and fail immediately.

### Fix

Compute the deadline independently per site master so each gets the full timeout window.

--- a/core/src/test/java/org/infinispan/xsite/AbstractXSiteTest.java
+++ b/core/src/test/java/org/infinispan/xsite/AbstractXSiteTest.java
@@ -151,7 +151,7 @@ public abstract class AbstractXSiteTest extends AbstractCacheTest {
     * Wait for all the site masters to see a specific list of sites.
     */
    public void waitForSites(long timeout, TimeUnit unit, String... siteNames) {
-      long deadlineNanos = System.nanoTime() + unit.toNanos(timeout);
+      long timeoutNanos = unit.toNanos(timeout);
       Set<String> expectedSites = new HashSet<>(Arrays.asList(siteNames));
       sites.forEach(site -> {
          site.cacheManagers.forEach(manager -> {
@@ -159,6 +159,7 @@ public abstract class AbstractXSiteTest extends AbstractCacheTest {
             RELAY2 relay2 = transport.getChannel().getProtocolStack().findProtocol(RELAY2.class);
             if (!relay2.isSiteMaster())
                return;
+            long deadlineNanos = System.nanoTime() + timeoutNanos;
             while (System.nanoTime() - deadlineNanos < 0) {
                if (expectedSites.equals(transport.getSitesView()))
                   break;

